emailvalidation/emailvalidation.go

```diff
@@ -0,0 +1,25 @@
+package emailvalidation
+
+import (
+	"net"
+	"regexp"
+	"strings"
+)
+
+var emailRegex = regexp.MustCompile("^[a-zA-Z0-9.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$")
+
+//IsEmailValid is a function to validate email addresses
```

Comment:
Probably worth adding what is considered a valid email to the GoDoc - helpful to anyone using this func.

Reply:
Documentation has been added to capture what has been considered a valid email